package.json

```diff
@@ -5,12 +5,19 @@
   "scripts": {},
   "private": true,
   "dependencies": {
+    "@leancode/cqrs-client-base": "^6.2.0-alpha.11",
+    "@nestjs/axios": "^0.0.5",
     "@nestjs/common": "^8.0.0",
+    "@nestjs/config": "^1.1.6",
     "@nestjs/core": "^8.0.0",
+    "@nestjs/passport": "^8.1.0",
     "@nestjs/platform-express": "^8.0.0",
     "core-js": "^3.6.5",
     "date-fns": "^2.28.0",
-    "puppeteer": ">=13.0.0",
+    "jwks-rsa": "^2.0.5",
+    "passport": "^0.5.2",
+    "passport-jwt": "^4.0.0",
+    "puppeteer": "^9.0.0",
```

Comment:
Why it got downgraded from `13`?

Reply:
because versions >= 10 use this https://github.com/advisories/GHSA-r683-j2x4-v87g